<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FocusControlWidgetClass</class>
 <widget class="QWidget" name="FocusControlWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>539</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>FocusControlWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_focus" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="grp_1dCommand_2">
        <property name="title">
         <string>Command</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="focus_move">
           <property name="text">
            <string>Move</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="focus_stop">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <item row="1" column="4">
         <widget class="QLabel" name="vid">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Position(mm):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="camera_pos">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>VID(mm)：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="4">
         <widget class="QPushButton" name="getPosition">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_14">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QGroupBox" name="grp_rel_2">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QRadioButton" name="focus_relative">
              <property name="text">
               <string>Relative</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="focus_global">
              <property name="text">
               <string>Absolute</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="focus_VID">
              <property name="text">
               <string>VID</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QGroupBox" name="grb_1DGlobal_2">
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QLabel" name="label_29">
                <property name="text">
                 <string>Relative(mm) :  </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="focus_Addition">
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="focus_relative_text"/>
              </item>
              <item>
               <widget class="QPushButton" name="focus_subtraction">
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_9">
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_30">
                <property name="text">
                 <string>Absolute(mm) :    </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="focus_global_text"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_16">
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <property name="spacing">
               <number>17</number>
              </property>
              <item>
               <widget class="QLabel" name="label_31">
                <property name="text">
                 <string>VID( mm ):      </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="focus_vid_text">
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
